spec/features/send_correction_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Send correction', type: :feature do
+  scenario 'A user can send a correction' do
+    VCR.use_cassette('send_correction') do
+      visit '/catalog/1002481'
+      within('.show-tools') do
+        click_link 'Send Correction'
+      end
+
+      fill_in 'correction[message]', with: 'correction message'
+      expect { click_button 'Submit Error, Typo or Suggestion' }
+        .to change { ActionMailer::Base.deliveries.count }.by(1)
+
+      expect(page).to have_content 'Thank you! We have received your correction and will be reviewing it soon.'
```

Comment:
I probably haven't been paying enough attention to this so there's cleanup to do elsewhere, but we could DRY this by using the I18n value instead of copying the expected text.
```suggestion
      expect(page).to have_content I18n.t('request_form_success')
```